src/ReverseImageSearchForm.js

```diff
@@ -1,18 +1,67 @@
 import React, { useRef } from "react";
 import Form from 'react-bootstrap/Form';
-
+import Col from 'react-bootstrap/Col';
+import { Slider, IconButton , Tooltip } from '@material-ui/core';
+import { MdInfo } from "react-icons/md";
 const ReverseImageSearchForm = params => {
 	const handleChange = (e) => {
-	  params.setState({file:e.target.files[0]});
+	  params.setFileState({file:e.target.files[0]});
 	  console.log({file:e.target.files[0]});
 	};
 
+	const marks = [
+		{
+			value: 0,
+			label: 'Very Low',
+		},
+		{
+			value: 25,
+			label: 'Low',
+		},
+		{
+			value: 50,
+			label: 'Medium',
+		},
+		{
+			value: 75,
+			label: 'High',
+		},
+		{
+			value: 100,
+			label: 'Very High',
```

Comment:
Please reverse the order of the labels. "very high" will return less results than "very low" so I'm hoping that having "very high" on the left and "very low" on right will indicate an increasingly larger set of results from left to right (for left to right readers). Does that make sense?

Reply:
Yes it does. I'll make the change.